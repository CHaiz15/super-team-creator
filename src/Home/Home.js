import React from 'react'
import './Home.scss'
import { NavLink } from 'react-router-dom'

export const Home = () => {
  return (        
    <main className="Home">
      <div>
      </div>
        <h1 className='main-title'>Super Team Creator</h1>
      <div className='instruction-box'>
        <h3 className='instructions-title'>Instructions</h3>
        <p className='instructions'>Have you ever wanted to create your own super team of heros and villians? Well now you can with the Super Team Creator! Just type in 3 of your favorite heros or villians. Give your team a heroic name and checkout what their stats are! Try all types of combinations and save your favorites!</p>
      </div>
      <NavLink type='button' to='/create' className='create-btn'>Create</NavLink>
    </main>
  )
}

export default Home;