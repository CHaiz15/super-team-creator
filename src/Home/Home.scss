@import '../variables';

.Home {
  height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;
}

.main-title {
  @include mainTitles();
  margin: -5% 0 20% 0;
  text-align: center;
}

.instruction-box {
  height: 30rem;
  background-color: white;
  border-radius: 10px;
  width: 50rem;
  display: flex;
  flex-direction: column;
  align-items: center;
  opacity: 0.95;
  animation: slide-top 0.5s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}

.instructions-title {
  font-size: 30px;
}

.instructions {
  margin: 0;
  font-size: 20px;
  width: 80%;
  line-height: 3rem;
}

.create-btn {
  @include mainBtn();
}

.create-btn:hover {
  @include mainBtnHover();
}

.create-btn:active {
  @include mainBtnActive();
}

a {
  text-decoration: none;
  list-style: none;
  color: black;
  text-align: center;
}

@keyframes slide-top {
  0% {
    transform: translateY(0);
  }
  100% {
    transform: translateY(-100px);
  }
}

@keyframes slide-bottom {
  0% {
    transform: translateY(0);
  }
  100% {
    transform: translateY(100px);
  }
}


