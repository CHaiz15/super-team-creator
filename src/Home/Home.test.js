import React from 'react';
import Home from './Home';
import { shallow } from 'enzyme'

describe('Home', () => {
  it('should be an instance of the Home component', () => {
    const wrapper = shallow(<Home/>);
      expect(wrapper).toMatchSnapshot();
  });
})
