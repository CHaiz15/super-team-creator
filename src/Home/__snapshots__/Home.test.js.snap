// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should be an instance of the Home component 1`] = `ShallowWrapper {}`;
